x-pack/plugins/enterprise_search/public/applications/applications/components/engine/engine_connect/search_application_documentation.tsx

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import welcomeGraphicLight from '../../../../../assets/images/welcome_light.svg';
+
+export const SearchApplicationDocumentation = () => {
+  return (
+    <EuiPanel color="transparent">
+      <EuiFlexGroup justifyContent="spaceBetween" direction="row">
+        <EuiFlexGroup direction="column">
+          <EuiImage
+            size="l"
+            float="left"
+            src={welcomeGraphicLight}
+            alt={i18n.translate(
+              'xpack.enterpriseSearch.content.searchApplication.documentation.welcomeGraphicLightImage.altText',
+              {
+                defaultMessage: 'Search Application documentaion illustration',
+              }
+            )}
+          />
+
+          <EuiText>
+            <h2>
+              {i18n.translate(
+                'xpack.enterpriseSearch.content.searchApplication.documentation.text.title',
+                {
+                  defaultMessage: 'Learn more about Search Applications',
+                }
+              )}
+            </h2>
+          </EuiText>
+
+          <EuiText>
+            <p>
+              {i18n.translate(
+                'xpack.enterpriseSearch.content.searchApplication.documentation.text.paragraph',
+                {
+                  defaultMessage:
+                    'Search Applications help make your Elasticsearch data searchable for end users, with out-of-the-box relevance, analytics and personalization tools.',
+                }
+              )}
+            </p>
+          </EuiText>
+
+          <EuiLink href="#" external>
+            {i18n.translate(
+              'xpack.enterpriseSearch.content.searchApplication.documentation.readMoreLink',
+              {
+                defaultMessage: 'Read our documentation',
```

Comment:
Question @julianrosado  Which documentation should these `Learn more` links take to?

Reply:
The doc links are all found here: https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/get_doc_links.ts#LL125C9-L125C9

_Enterprise Search:_

- Search Applications docs
- Behavioral Analytics doc

_Kibana_:
- API key docs

_ES language clients docs_
- Clients guide

Reply:
But not clear if we want all of these, let's continue convo in [Slack](https://elastic.slack.com/archives/CHY7020LT/p1683825097672239?thread_ts=1683825014.626999&cid=CHY7020LT).

Reply:
All good for these links, just some copy changes to avoid confusion!